templates/bs-table-transactions.phtml

```diff
@@ -77,9 +77,14 @@
 </tr>
 {/foreach}
 {/if}
+<tr>
+	<td colspan=3><a href='{$thisScript}?verb=moreTransactions&windowEndDate={$nextWindowEnd}' data-target='#moreTransactions'>Load more transactions</a>
+</tr>
 </tbody>
 </table>
 
+<div id='moreTransactions'></div>
```

Comment:
this is completely unused and isn't the way this should go.  i should probably add a class to the above table row (and all the uncleared transactions), making it selectable so i can wipe it and replace it with the contents of an ajax request.